### DISCUSSION TITLE
Fehler beim pushen der Firmware

### DISCUSSION THREAD

**Discussion by adorableAppa** [Freetz-NG/freetz-ng#1079] [Allgemein]
Huhuu, 
ich möchte mich auch gerne an Freetz versuchen und habe die passende Firmware builden können und habe auch eine .image erhalten. 
Ich versuchte mit Befehlen wie:
`freetz@freetz:~freetz-ng/tools$ ./push_firmware /mnt/shared/firmware.image`
und
`freetz@freetz:~freetz-ng$ tools/push_firmware /home/freetz/freetz-ng/images/6590_07.57.ger_.....`

Jedoch erhalte ich immer die Fehlermeldung beim Pushen
```
local: mtd remote: mtd
local: mtd: No such file or directory
```

Ich habe euch mal ein Bild angehangen. :)

Ich bedanke mich an dieser Stelle im Voraus für die Hilfe!

![grafik](https://github.com/user-attachments/assets/cfd0c374-3b12-4630-a3f3-1830b50934fc)

**Comment by fda77** [comment]
sieht doch gut aus. 
diese "local" kommen wohl von deinem ftp programm. du kannst due befehle auch von hand eingeben
was ist denn der fehler?

**Reply by adorableAppa** [reply]
Na ja, ich weiß nicht, ob Freetz jetzt überhaupt installiert ist. Sehen tu' ich davon nichts.

**Reply by fda77** [reply]
na wenn du noch nicht mal weisst ob es überhaupt ein problem gibt

**Reply by adorableAppa** [reply]
Na ja, durch den localen Fehler und dadurch, dass ich im Router nichts sehe, gehe ich davon aus, dass die Übertragung erfolglos war.

**Reply by pfichtner** [reply]
> Na ja, durch den localen Fehler und dadurch, dass ich im Router nichts sehe, gehe ich davon aus, dass die Übertragung erfolglos war.

Die Hardware im Router ändert sich nicht, im Router wirst Du daher nix sehen
SCNR!

https://freetz-ng.github.io/freetz-ng/wiki/10_Beginner/newbie.de.html
(http://fritz.box:81 alternativ IP-Adresse)
